core-lib/src/main/java/org/idpass/smartscanner/lib/scanner/config/NFCOptions.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Newlogic Pte. Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *
+ */
+package org.idpass.smartscanner.lib.scanner.config
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class NFCOptions(
+    val locale: String? = null,
+    val withPhoto: Boolean? = null,
+) : Parcelable {
+    companion object {
+        val default = NFCOptions(
+            withPhoto = true
```

Comment:
I just like to understand, a little bit, how a true/false configuration is set. So, it is set to `true` or `false` in this file? And nowhere else, such as in `build.gradle` or `local.properties` for example. Thanks..

Reply:
So there is separate `NFCOptions.kt` for `Tasjael` where `withPhoto = false` ?

Reply:
@typelogic for our app default for `withPhoto = true` and can be accessed via 

`MainActivity` class starting at Line 179
See: https://github.com/idpass/smartscanner-core/blob/feature/nfc-config/app/src/main/java/org/idpass/smartscanner/MainActivity.kt#L179

Then to access it via smartscanner-capacitor like for `Tasjael`. 
It is called via method:
```
const result = await SmartScannerPlugin.executeScanner({
  action: 'START_SCANNER',
  options: {
    mode: 'nfc-scan',
    nfcOptions: {
      withPhoto: false
    }
    config: {
      background: '#89837c',
      branding: false,
      isManualCapture: false,
    },
  },
});
```